OrbitGl/TimeGraph.h

```diff
@@ -137,8 +144,8 @@ class TimeGraph {
   double m_RefTimeUs = 0;
   double m_MinTimeUs = 0;
   double m_MaxTimeUs = 0;
-  TickType m_SessionMinCounter = 0;
-  TickType m_SessionMaxCounter = 0;
+  TickType m_CaptureMinCounter = 0;
```

Comment:
Should we use the new style capture_min_counter_ ?

Reply:
Yes, of course, thanks!